resources/rafter/charts/upload-service/templates/destination-rule.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: "networking.istio.io/v1alpha3"
```

Comment:
Please add this template to this https://github.com/kyma-project/kyma/pull/6468 PR, or if you don't have a permission to do this, please write in comment and I will do it for you.

Reply:
This patch is for `rafter` umbrella chart that is already in master. 
We do not need to wait on #6468 to be merged.